llama_stack/apis/common/content_types.py

```diff
@@ -21,7 +20,8 @@ class URL(BaseModel):
 
 class _URLOrData(BaseModel):
     url: Optional[URL] = None
```

Comment:
can you add a doc string inside this class too? that should get parsed (I think) and get represented in the spec

Reply:
done